docs/tutorials/03_european_call_option_pricing.ipynb

```diff
@@ -399,14 +394,13 @@
     "epsilon = 0.01\n",
     "alpha = 0.05\n",
     "\n",
-    "qi = QuantumInstance(Aer.get_backend(\"aer_simulator\"), shots=100)\n",
     "problem = EstimationProblem(\n",
     "    state_preparation=european_call,\n",
     "    objective_qubits=[3],\n",
     "    post_processing=european_call_objective.post_processing,\n",
     ")\n",
     "# construct amplitude estimation\n",
-    "ae = IterativeAmplitudeEstimation(epsilon, alpha=alpha, quantum_instance=qi)"
+    "ae = IterativeAmplitudeEstimation(epsilon_target=epsilon, alpha=alpha, sampler=Sampler())"
```

Comment:
The qi before had shots=100. I see below the sampler was set with shots=100 in run_options. But not here?